pyogrio/tests/test_geopandas_io.py

```diff
@@ -62,6 +62,25 @@ def spatialite_available(path):
         return False
 
 
+def test_read_csv_encoding(tmp_path):
+    # Write csv test file. Depending on the os this will be written in a different
+    # encoding: for linux and macos this is utf-8, for windows it is cp1252.
```

Comment:
In one of the next Python versions, this will become utf-8 for windows as well. Do we want to have an additional test where we manually specify the encoding to test a specific one?

Reply:
OK, the test now tests encodings "UTF-8", "cp1252" and None.